storage/storage.go

```diff
@@ -1704,6 +1710,8 @@ type Conditions struct {
 	// GenerationNotMatch specifies that the object must not have the given
 	// generation for the operation to occur.
 	// If GenerationNotMatch is zero, it has no effect.
+	// This condition only works for object reads if the WithJSONReads client
```

Comment:
`[WithJSONReads]` to make a link, I think

Reply:
I double checked this, it does not create a link in VSCode nor on pkg.go.dev docs. See attachment.

<img width="732" alt="image" src="https://user-images.githubusercontent.com/29010659/215636319-b060f630-970c-4cba-880c-065f6e533b79.png">